### PR TITLE
New baselines for CSharpReservedWordsTest, CSharpWhitespaceHandlingTe…

### PR DESCRIPTION
**Pull request by ajaybhargavb**
…st and CSharpSpecialBlockTest

#2698

### CODE REVIEW

File: test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpReservedWordsTest/ReservedWordIsCaseSensitive.stree.txt [hunk 1]
```diff
@@ -1,3 +1,8 @@
-Expression block - Gen<Expr> - 9 - (0:0,0)
-    Code span - Gen<Expr> - [NameSpace] - ImplicitExpressionEditHandler;Accepts:NonWhitespace;ImplicitExpression[RTD];K14 - (0:0,0) - Tokens:1
-        SyntaxKind.Identifier;[NameSpace];
+CSharpCodeBlock - [0..9)::9 - [NameSpace]
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpReservedWordsTest/ReservedWordIsCaseSensitive.stree.txt]: I suck, why did I add tests like this lol

File: test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpSpecialBlockTest/ParseBlockTerminatesSingleLineCommentAtEndOfLine.stree.txt [hunk 2]
```diff
@@ -1,38 +1,40 @@
...
+        Keyword;[if];
+        LeftParenthesis;[(];
+        Not;[!];
+        Keyword;[false];
+        RightParenthesis;[)];
+        Whitespace;[ ];
+        LeftBrace;[{];
+        NewLine;[LF];
+        Whitespace;[    ];
+        CSharpComment;[// Foo];
```
Comment (NTaylorMullen) [inline test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpSpecialBlockTest/ParseBlockTerminatesSingleLineCommentAtEndOfLine.stree.txt]: jesssus lol, so much knowledge!

Comment (NTaylorMullen) [reply test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpSpecialBlockTest/ParseBlockTerminatesSingleLineCommentAtEndOfLine.stree.txt]: Curious: Do we understand `/* */` as well?

Comment (ajaybhargavb) [reply test/Microsoft.AspNetCore.Razor.Language.Test/TestFiles/ParserTests/CSharpSpecialBlockTest/ParseBlockTerminatesSingleLineCommentAtEndOfLine.stree.txt]: No. Just the `// ` one.

**Comment by ajaybhargavb** [comment]
Merged e15971e9eb2b3734aa2c6e270edbd4b119441935

**Pull request closed without merging**